src/components/QuestionThree/QuestionThree.css

```diff
@@ -0,0 +1,39 @@
+.answer-options {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 10px;
+  margin-top: 10px;
+}
+
+.answer-label {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  padding: 10px;
+  font-size: 16px;
+  color: black;
```

Comment:
can you change it to some other color? the contract between the background color and this isn't very distinguishable